tests/unit/htdocs/Frameworks/moduleclasses/ModuleAdminTest.php

```
@@ -0,0 +1,429 @@
+    {
+        // Set up a mock XoopsModule in globals
+        $GLOBALS['xoopsModule'] = $this->createMockXoopsModule();
+        $GLOBALS['xoopsConfig'] = ['language' => 'english'];
+
+        $this->admin = new ModuleAdmin();
+    }
+
+    protected function tearDown(): void
+    {
+        unset($GLOBALS['xoTheme']);
+    }
```

PHPMD (2.15.0)

[error] tearDown accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)